docs/community_resources/support_maintenance_faq.md

```diff
@@ -39,29 +39,20 @@ To read more about language managing in eZ Platform, see the following doc pages
 - [Back Office languages](https://doc.ezplatform.com/en/latest/guide/internationalization/#back-office-languages)
 - [Setting up multi-language SiteAccesses and corresponding translations](https://doc.ezplatform.com/en/latest/cookbook/setting_up_multi_language_siteaccesses/)
 
-#### How can I contribute code to eZ Platform?
+#### How can I apply patches to the installation?
 
-First, create an issue in our [issue tracker](https://jira.ez.no/browse/EZP) and refer to it in commits and pull request headers.
-For example: `EZP-20104: ContentController should return error status when content is not found`.
-Then, you can fork our repository and create your own branch from the version of the package which you want to contribute to.
-
-Now introduce your changes, either by modifying existing files or creating new ones.
-Once you are done with your edits, add your files to the staging area,
-then commit with a short, clear description and push changes to your fork.
-Finally, go to the project's page on GitHub and you should see a `Compare and pull request` button.
-Click it, write a description and select `Create pull request`.
-Start the pull request's name with the issue number.
-Now you can wait for your changes to be reviewed and eventually tested and merged into the product.
+The easiest way to apply patch to your project is using unix [patch](http://man7.org/linux/man-pages/man1/patch.1.html) command. You should execute it from the directory which is dedicated for particular package e.g. `vendor/ezsystems/ezplatform-admin-ui`. Before running it, we advise to test the patch file using `--dry-run` parameter to see what files will be modified. It is also good to prepare a backup of those files. Below there is an example call.
```

Comment:
do we need to cover use of patch?
1. this people can figure out themselves if they don't know how to deal with patches. 2. the real value comes when using composer-patches as it then has conventions that allows you to check it in to your code of your project code _(root meta repo based)_, and drop it once you do composer update and pull in release that already has the patch (in the same commit)

Reply:
@andrerom please take a look at https://github.com/ezsystems/developer-documentation/pull/616/commits/c0735ba0bf09b42309df500d7972db473c4526d6 and let me know if this is what you have in mind.